src/Type.php

```diff
@@ -223,6 +255,16 @@ public function isSubtypeOf(self $other): bool
                 }
             }
         }
+        if ($this->name === 'Struct') {
```

Comment:
`$this->isStruct()`?

Reply:
See https://github.com/eventjet/ausdruck/pull/25#discussion_r1740649431